src/render.js

```diff
@@ -0,0 +1,11 @@
+import leaderBoard from './leaderBoard.js';
+import { renderSection } from './variables.js';
+
+async function render() {
```

Comment:
- [ ] Kindly use  arrow functions instead of the `function` keyword as this is part of the project requirements 👍🏿